docs/utilities.rst

```diff
@@ -80,6 +80,13 @@ you'll need to have it start in the background -- something you'll need to
 figure out for your own system.  To get you started though, there are Systemd
 service files in the ``scripts`` directory.
 
+Some command line argument are available to customize the behavior of the
```

Comment:
The only thing I could find that needs attention is this typo here :-)  It should probably be `Some comand line arguments`.